automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -421,21 +439,30 @@ def sync_capsule_repos_to_upgrade(capsules):
         '{} '.format(cv_name))[1]) for data in cv_version_data]).pop()
     cv_ver_id = get_attribute_value(cv_version_data, '{0} {1}'.format(
         cv_name, latest_cv_ver), 'id')
-    hammer_content_view_promote_version(cv_name, cv_ver_id, lc_env_id, '1')
-    # Add new product subscriptions to AK
-    hammer_activation_key_add_subscription(ak_name, '1', 'capsule6_latest')
+    hammer_content_view_promote_version(
+        cv_name, cv_ver_id, lc_env_id, '1')
+    # If Downstream, Add new product subscriptions to AK
```

Comment:
In case custom capsule repo URL given, update AK with capsule product subscription.